lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/NodeStruct.cs

```diff
@@ -36,7 +41,7 @@ internal class NodeStruct
         public NodeStruct(string id)
         {
             Identifier = id;
-            _messageQueue = new BlockingCollection<GroupCommunicationMessage>();
+            _messageQueue = new BlockingCollection<GroupCommunicationMessage<T>>();
```

Comment:
I think u r saying this one shud be GeneralGroupCommunicationMessage. See my answer above.